core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/function/FunctionExpression.kt

```diff
@@ -9,18 +9,18 @@ import ch.ergon.dope.validtype.ValidType
 
 abstract class FunctionExpression<T : ValidType>(
     private val symbol: String,
-    private vararg val expressions: UnaliasedExpression<T>,
+    private vararg val expressions: UnaliasedExpression<out ValidType>?,
 ) : TypeExpression<T>, FunctionOperator {
     override fun toDopeQuery(manager: DopeQueryManager): DopeQuery {
-        val expressionsDopeQuery = expressions.map { it.toDopeQuery(manager) }
+        val expressionsDopeQuery = expressions.map { it?.toDopeQuery(manager) }
```

Comment:
would use the method _mapNotNull_ instead of _map_, then you get rid of the _orEmpty()_ below

Reply:
done